lib/thor/actions/file_manipulation.rb

```diff
@@ -117,7 +117,13 @@ def template(source, *args, &block)
       context = config.delete(:context) || instance_eval("binding")
 
       create_file destination, nil, config do
-        content = CapturableERB.new(::File.binread(source), nil, "-", "@output_buffer").tap do |erb|
+        match = ERB.version.match(/(\d\.\d\.\d)/)
```

Comment:
This won't work on versions with more numerics. While it's not a problem now, it could cause wierd errors in the future.

example
```ruby
'2.11.0'.match(/(\d\.\d\.\d)/) -> nil
```
an old behaviour is used even if '2.11.0' is newer than '2.2.0'
```ruby
'2.11.0' >= '2.2.0' -> false
```